### PR TITLE
Update to Abp 5.6 & Fix Swagger UI for Kestrel web server

### PR DESCRIPTION
**Pull request by do-it-ger**
Update to Abp 5.6

### CODE REVIEW

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Host/Startup/Program.cs [hunk 1]
```diff
@@ -12,8 +14,14 @@ public static void Main(string[] args)
...
         public static IWebHost BuildWebHost(string[] args)
         {
+            var config = new ConfigurationBuilder()
+               .SetBasePath(Directory.GetCurrentDirectory())
+               .AddJsonFile("appsettings.json", optional: false)
+               .Build();
+
             return WebHost.CreateDefaultBuilder(args)
                 .UseStartup<Startup>()
+                .UseUrls(config.GetValue<string>("App:ServerRootAddress"))
```
Comment (maliming) [inline aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Host/Startup/Program.cs:24]: hi
Set the url in `appsettings.json` will be clearer.
```json
  "Kestrel": {
    "Endpoints": {
      "Http": {
        "Url": "http://localhost:21021/"
      }
    }
  }
```

Comment (do-it-ger) [reply aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Host/Startup/Program.cs:24]: I can do it. The idea was to use only one webserver config for kestrel and CORS.

**Comment by do-it-ger** [comment]
Add Kestrel config to appsettings.json

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Host/Startup/Program.cs [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ﻿using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
```
Comment (ismcagdas) [inline aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Host/Startup/Program.cs:3]: Using statements seems unnecessary.

**Comment by ismcagdas** [comment]
Thanks a lot @do-it-ger 😄

**Pull request merged**